lib/gooddata/models/user_filters/mandatory_user_filter.rb

```diff
@@ -28,7 +28,7 @@ def all(options = { client: GoodData.connection, project: GoodData.project })
           result = c.get("/gdc/md/#{project.pid}/userfilters?count=#{count}&offset=#{offset}")
           result['userFilters']['items'].each do |item|
             item['userFilters'].each do |f|
-              user_lookup[f] = item['user']
+              user_lookup[f] = user_lookup[f] ? Array(user_lookup[f]).concat(Array([item['user']])) : item['user']
```

Comment:
Could `Array([item['user']])` be shortened to `[item['user']]`?

Reply:
no, as [[1]] = [[1]] but Array([1]) = [1]

Reply:
good idea, fixed